components/server/ee/src/prebuilds/prebuilt-status-maintainer.ts

```diff
@@ -197,15 +199,11 @@ export class PrebuildStatusMaintainer implements Disposable {
                                 : "success",
                     });
                 } catch (err) {
-                    if (err.message == "Not Found") {
-                        log.info(
-                            "Did not find repository while updating updatable. Probably we lost the GitHub permission for the repo.",
-                            { owner: updatable.owner, repo: updatable.repo },
-                        );
-                        found = true;
-                    } else {
-                        throw err;
-                    }
+                    log.info("Could not create commit status.", { updatable, errorMessage: err?.message });
+                    // e.g. this happens if a commit is removed from remote repository (force push)
+
+                    // let's just assume, we have not found the commit to create a status for
+                    found = false;
```

Comment:
Do we need to keep it if we're not sure what it means? E.g., either we keep the extra-case for "Not found" and report "found", or we drop it altogether.
The span would benefit from a `TraceContext.setError({ span }, err);`, which is now not done anymore below.